src/main/plugins/org.dita.eclipsehelp/build_dita2eclipsehelp_template.xml

```diff
@@ -26,14 +29,21 @@ See the accompanying LICENSE file for applicable license.
 	
     <target name="dita2eclipsehelp"
             unless="noMap"
-        depends="build-init, dita.eclipsehelp.init, preprocess, 
+        depends="dita.eclipsehelp.init, build-init, preprocess, 
                  xhtml.topics,
                  copy-css">
       <antcall target="dita.map.eclipse"></antcall>
+      <jar destfile="${output.dir}/${args.eclipsehelp.jar.name}" basedir="${dita.output.dir}" if:set="args.eclipsehelp.jar.name">
```

Comment:
Antcall was probably used because the original coder didn't know there was any difference between antcall and using a dependency (antcall creates a new Project and thus allows overriding property values).

In this case you can just move `dita.map.eclipse` to dependencies.